### PR TITLE
Added ip address script

### PR DESCRIPTION
**Pull request by codewithpom**

### CODE REVIEW

File: ip/ip.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+curl ipinfo.io/json -q | jq
```
Comment (suraj-singh12) [inline ip/ip.sh:1]: Fails with errors:
`jq: command not found`
`curl: (23) Failed writing body`

**Review by suraj-singh12 — CHANGES_REQUESTED** [review]

File: ip/ip.sh (hunk 1 above)
Comment (codewithpom) [reply ip/ip.sh:1]: You need to install jq for that by 
```bash
sudo apt-get install jq
```

Comment (suraj-singh12) [reply ip/ip.sh:1]: Add that to the script. Other users might not have it installed, just like I don't.
Your script should work out of the box.

Comment (codewithpom) [reply ip/ip.sh:4]: How is it now ?

**Review by suraj-singh12 — APPROVED** [review]

**Pull request merged**